warehouse/models/mart/transit_database/dim_organizations.sql

```diff
@@ -20,6 +20,7 @@ dim_organizations AS (
         dim.source_record_id,
 
         ntd.key AS annual_database_agency_information_2021_key,
+        ntd.ntd_id,
```

Comment:
Can you just use `ntd_to_org.ntd_id` here? Do we even need the second join?

Reply:
The join was redundant, so was removed in [84691c6](https://github.com/cal-itp/data-infra/pull/2582/commits/84691c609797dde00a57a56a90e3b0e939a7442d).